components/Play.tsx

```diff
@@ -26,14 +28,18 @@ const Play = ({ data }: PlayProps) => {
     const nextIndex = currentIndex + 1;
     setIsFinished(nextIndex >= data.words.length);
 
+    if (!isCorrect) return;
```

Comment:
@urassh 

次の単語に進んだ後、フリック入力をせずにEnterKeyを押すと次の単語に進めてしまうバグがありました！
他に良い解決方法がある気がしますが、ここでフラグを`false`にすると一応は防げます。

```suggestion
    if (!isCorrect) return;
    setIsCorrect(false);
```

Reply:
@shokm 
うっかりしてました。。。!!
それが一番良い気がするので、修正やっておきました!!